src/slices/gateArtifactsSlice.ts

```diff
@@ -49,18 +51,24 @@ export interface StateGatingTests {
     sstTeams: string[];
 }
 
+export type StateGatingTestsSerialized = Partial<
+    Pick<StateGatingTests, 'ciSystem' | 'gateTag' | 'packager'> & {
+        btype: keyof typeof BUILD_TYPE_MENU_ITEMS;
+        ic: string;
+        pid: string;
+        sstTeams: string;
+    }
+>;
+
 const INITIAL_STATE: StateGatingTests = {
-    buildType: getTypeFromSelect(query.get('btype') || 'ordinary'),
-    ciSystem: decodeURIComponent(query.get('cisystem') || ''),
-    gateTag: decodeURIComponent(query.get('gatetag') || ''),
-    ignoreCiSystem: decodeURIComponent(query.get('ic') || 'false') === 'true',
-    packager: decodeURIComponent(query.get('packager') || ''),
-    productId: _.toInteger(query.get('pid') || 604),
+    buildType: 'brew-build',
```

Comment:
@mgrabovsky how do the URL args will work?

Reply:
They are handled in `PageGating` via React Router's `useSearchParams`, which is a sort of a state in itself.